docroot/themes/contrib/civictheme/civictheme_library/components/01-atoms/chip/chip.js

```diff
@@ -88,8 +96,10 @@ CivicChip.prototype.focusoutEvent = function (e) {
 CivicChip.prototype.dismissClickEvent = function (e) {
   const chip = this.findChip(e.target);
   if (chip) {
-    chip.remove();
-    this.el.dispatchEvent(new CustomEvent('civictheme.chip.dismiss', { bubbles: true }));
+    if (!chip.classList.contains('civictheme-chip--input')) {
```

Comment:
please do not use presentation class to determine the kind of the Chip. Please create a helper function that would be assessing if this element is an `<input>` and would return a kind.